libs/application/templates/children-residence-change/src/fields/ContractRejected/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { contractRejected } from '../../lib/messages'
+import { CRCFieldBaseProps } from '../../types'
+import { ContractRejectedContainer, DescriptionText } from '../components'
+import { getSelectedChildrenFromExternalData } from '../..'
+
+const ContractRejected = ({ application }: CRCFieldBaseProps) => {
+  const { answers } = application
+  const selectedChildren = getSelectedChildrenFromExternalData(
+    application.externalData.nationalRegistry.data.children,
+    answers.selectedChildren,
+  )
+
+  const otherParentName = selectedChildren[0].otherParent.fullName
```

Comment:
Not a part of this PR but I'm just wondering if we cant to create a utility for extracting `otherParent` from the application. Feel like we are doing this in a lot of places where we are using the `selectedChildren` only to get the `otherParent`.

Reply:
Definitely, I was also wondering if we could get this any other way when I was writing this but didn't see another way, i'll add a refactor ticket to our board. The ticket: https://app.asana.com/0/1199733748699731/1200295507610131/f